telegram/_files/inputsticker.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2023
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains an object that represents a Telegram InputSticker."""
+
+from typing import Optional, Sequence, Tuple, Union
+
+from telegram._files.inputfile import InputFile
+from telegram._files.sticker import MaskPosition
+from telegram._telegramobject import TelegramObject
+from telegram._utils.argumentparsing import parse_sequence_arg
+from telegram._utils.files import parse_file_input
+from telegram._utils.types import FileInput, JSONDict
+
+
+class InputSticker(TelegramObject):
+    """
+    This object describes a sticker to be added to a sticker set.
+
+    Objects of this class are comparable in terms of equality. Two objects of this class are
+    considered equal, if their :attr:`sticker` and :attr:`emoji_list` are equal.
```

Comment:
Do we actually have meaningful comparison of `InputFile`? If not, then I would vote to just not make `InputSticker` comparable … How do we do that for `InputMedia*`?

Reply:
you're right, we don't have equality for InputFile and InputMedia objects are not comparable